modules/support-get-cluster-id.adoc

```diff
@@ -5,21 +5,30 @@
 [id="support-get-cluster-id_{context}"]
 = Obtaining your cluster ID
 
-When providing information to Red Hat Support, it is helpful to provide the unique identifier for your cluster. You can obtain your cluster ID by using the OpenShift CLI (`oc`) or by using the {product-title} web console.
+When providing information to Red Hat Support, it is helpful to provide the unique identifier for your cluster. You can have your cluster ID autofilled by using the {product-title} web console. You can also manually obtain your cluster ID by using the web console or the OpenShift CLI (`oc`).
 
 .Prerequisites
 
 * Access to the cluster as a user with the `cluster-admin` role.
-* The {product-title} CLI (`oc`) installed.
+* Access to the web console or the OpenShift CLI (`oc`) installed.
 
 .Procedure
 
+* To open a support case and have your cluster ID autofilled using the web console:
```

Comment:
Support Case here need to be conditionalized.

Reply:
Ah, good point. I updated conditionals so "support case" shows for Enterprise and "bug" shows for Origin.